c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -261,6 +261,8 @@ help:
 	@echo "- compile-py-catalog		Compile the angular translation catalog"
 	@echo "- flake8					Run flake8 checker on the Python code"
 	@echo "- lint-ngeo				Check the JavaScript code with linters for ngeo"
+	@echo "- template-clean		 Clean the template file"
```

Comment:
Shouldn't the indent be fixed in this section? (Command descriptions should be aligned?)